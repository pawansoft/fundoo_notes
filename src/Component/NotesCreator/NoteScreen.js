import React from 'react';
import NotesHolder from './NotesHolder';

const NoteScreen = ({navigation}) => {
    return(
        <NotesHolder navigation = {navigation}/>
    )
}
export default NoteScreen;