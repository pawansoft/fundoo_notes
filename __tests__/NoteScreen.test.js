import React from 'react';
import{configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import NoteScreen from '../src/Component/NotesCreator/NoteScreen';

configure({adapter: new Adapter()})

jest.mock('react-native-localization', () => class RNLocalization {
    language = 'en-US'
  
    constructor (props) {
      this.props = props
      this.setLanguage(this.language)
    }
  
    setLanguage (interfaceLanguage) {
      this.language = interfaceLanguage
      if (this.props[interfaceLanguage]) {
        var localizedStrings = this.props[this.language]
        for (var key in localizedStrings) {
          if (localizedStrings.hasOwnProperty(key)) this[key] = localizedStrings[key]
        }
      }
    }
})

describe('Test note screen', () => {
    it('ProvideNoteScreen_WhenSnapshotMatches_ShouldPassTestCase', () =>{
        const component = shallow(<NoteScreen/>)
        expect(component).toMatchSnapshot();
    })
})