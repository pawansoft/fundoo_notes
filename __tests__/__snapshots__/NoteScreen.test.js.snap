// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test note screen ProvideNoteScreen_WhenSnapshotMatches_ShouldPassTestCase 1`] = `ShallowWrapper {}`;
